packages/compute-baseline/src/browser-compat-data/supportStatements.ts

```diff
@@ -33,23 +33,35 @@ export class SupportStatement {
     this.feature = feature;
   }
 
+  /**
+   * An array of `FlagStatement` objects. If there are no flags, then the array
+   * is empty.
+   */
   get flags(): FlagStatement[] {
     return this.data?.flags ?? [];
   }
 
+  /**
+   * A `true` value, if support exists but may be incompatible with other
```

Comment:
Trying to summarize https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#partial_implementation-general-usage-guidelines in a single sentence is hard, but I think that "incompatible with other implementations" is one of the less common reasons. Most common is, I think, large deviations from the spec.

No change is really necessary, but if you feel inspired to change something, please go ahead.

Reply:
Good point. I applied d59c16e1, which describes partials as "irregular" (when I really wanted to say "weird" or "strange" but showed some restraint).